module/PowerShellEditorServices/PowerShellEditorServices.psm1

```diff
@@ -106,12 +112,24 @@ function Start-EditorServicesHost {
 
     if ($LanguageServiceNamedPipe) {
```

Comment:
If we used parameter sets then here we'd check if ($PSCmdlet.ParameterSetName -eq 'NamedPipeDuplex') and perhaps $LanguageServicePipe would be a mandatory parameter? Not sure about $DebugLanguageServicePipe, I suppose you could create one but not the other?